src/main/java/xyz/deseteral/lebkuchenfm/api/commands/text/TextCommandController.java

```diff
@@ -0,0 +1,39 @@
+package xyz.deseteral.lebkuchenfm.api.commands.text;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import xyz.deseteral.lebkuchenfm.domain.CommandProcessingResponse;
+import xyz.deseteral.lebkuchenfm.domain.CommandExecutor;
+import xyz.deseteral.lebkuchenfm.domain.TextIsNotACommandException;
+import xyz.deseteral.lebkuchenfm.domain.NoSuchCommandProcessorException;
+
+@RestController
+final class TextCommandController {
+    private final CommandExecutor processor;
+
+    TextCommandController(CommandExecutor processor) {
+        this.processor = processor;
+    }
+
+    @PostMapping("/commands/text")
+    public TextCommandResponseDto processCommand(@RequestBody TextCommandRequestDto commandDto) {
+        CommandProcessingResponse processingResponse = processor.processFromText(commandDto.getText());
+        return new TextCommandResponseDto(processingResponse);
+    }
+
+    @ExceptionHandler(NoSuchCommandProcessorException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public TextCommandResponseDto noSuchCommandExceptionHandler(NoSuchCommandProcessorException ex) {
+        return new TextCommandResponseDto(ex);
+    }
+
+    @ExceptionHandler(TextIsNotACommandException.class)
```

Comment:
Jakbyś chciał to można kontrolę wyjątków wyrzucić do osobnego handlera, ale to nic blokującego :)